### PR TITLE
A few minor improvements

### PR DESCRIPTION
**Pull request by bocharsky-bw**

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
👍

File: doctrine/associations.rst [hunk 1]
```diff
@@ -365,7 +365,7 @@ Of course, if you know up front that you'll need to access both objects, you
 can avoid the second query by issuing a join in the original query. Add the
 following method to the ``ProductRepository`` class::
 
-    // src/AppBundle/Entity/ProductRepository.php
+    // src/AppBundle/Repository/ProductRepository.php
```
Comment (bocharsky-bw) [inline doctrine/associations.rst]: I think to keep entities and repositories separately is a better option. Let's use the `Repository` namespace like in best practices. I'll push more changes

**Comment by xabbuh** [comment]
👍

**Comment by wouterj** [comment]
👍

**Comment by wouterj** [comment]
Thanks a lot for fixing this inconsistency in the docs, Victor! It's now merged in.

**Pull request merged**